packages/core/src/client.ts

```diff
@@ -71,7 +71,9 @@ export async function createClient (
       txBuffer?.push(tx)
     } else {
       hierarchy.tx(tx)
-      void model.tx(tx)
+      if (tx.objectSpace === core.space.Model) {
+        void model.tx(tx)
```

Comment:
Please also do same on line 55 ^^^

```typescript
 async tx (tx: Tx): Promise<void> {
    await this.conn.tx(tx)
    // Should also have
    hierarchy.tx(tx)
    if (tx.objectSpace === core.space.Model) {
       void model.tx(tx)
    }
}
```

Reply:
No, because we will have double tx in this case, conn.tx() will call handler....